src/components/ManageProjects.js

```diff
@@ -0,0 +1,127 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { deleteProject, updateProjectDescription, updateBeginningDate, updateDueDate } from './../actions';
+
+class ManageProjects extends React.Component{
+  removeProject = (e) => {
+    const projectInfo = "title=" + this.props.task.title
+    this.props.onDelete(this.props.taskId, projectInfo)
+    e.preventDefault();
+  }
+
+  editProjectDescription = (e) => {
+    if(this.editedDescription.value === '')
+      e.preventDefault();
+    else {
+      const projectInfo = "title=" + this.props.task.title + "&description=" + this.editedDescription.value
```

Comment:
You can have some utility method to create the bodyData as you need (here query string).